docs/css/github-permalink-style.css

```diff
@@ -37,3 +37,7 @@
 		margin-left: calc(var(--permalink-size) * -1 - var(--permalink-spacing)) !important;
 	}
 }
+
+.md-nav__icon {
+	margin-bottom: 0.5rem;
+}
```

Comment:
I am not sure if this is the case, but margins are off now in docs:
<img width="258" height="413" alt="image" src="https://github.com/user-attachments/assets/3ab8d219-fde2-4af3-9280-4cf63653c78e" />

It is ok in prod:
<img width="258" height="284" alt="image" src="https://github.com/user-attachments/assets/82631bb8-1ac3-4d5d-be65-5d5177351b2a" />

Reply:
Oh, thats weird, production in mine was different and not showing the margin in firefox. Let me revert this change then